src/pages/my-page/MypageEditPage.tsx

```
@@ -1,41 +1,100 @@
+    // 태그 데이터 포맷팅
+    const allTags = tagList?.data?.flatMap(cat => 
+        cat.tags.map(tag => ({
+            id: tag.id,
+            name: tag.name,
+        }))
+    ) || [];
+
+    const tagIdToName = useMemo(() => {
+        const m = new Map<number, string>();
+        allTags.forEach(t => m.set(t.id, t.name));
+        return m;
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)